include/autoppl/expression/variable/constant.hpp

```diff
@@ -11,12 +12,20 @@ struct Constant : util::VarExpr<Constant<ValueType>>
     Constant(value_t c)
         : c_{c}
     {}
-    value_t get_value(size_t) const {
+    value_t get_value(size_t = 0) const {
```

Comment:
Defaulting this scares me a little because it changes the API. Is there any way this could be a problem?

Reply:
We're ignoring this param anyway. This allows users to simply call `get_value()`. I think enforcing developers to pass in a number to `get_value` even when it's not going to get used is very confusing.